std_msgs/Float64 x1
std_msgs/Float64 y1
std_msgs/Float64 x2
std_msgs/Float64 y2